lib/fog/compute/google/requests/insert_disk.rb

```diff
@@ -29,14 +29,14 @@ def insert_disk(disk_name, zone_name, image_name = nil, options = {})
           data[:disks][disk_name] = object
 
           if image
-            object.merge!("sourceImage" => image.self_link,
-                          "sourceImageId" => image.id)
-          end
-          if disk_type = options.delete(:type)
-            object["type"] = type
-          else
-            object["type"] = "https://www.googleapis.com/compute/#{api_version}/projects/#{@project}/zones/#{zone_name}/diskTypes/pd-standard"
+            object["sourceImage"] = image.self_link
+            object["sourceImageId"] = image.id
           end
+          object["type"] = if disk_type = options.delete(:type)
```

Comment:
@icco Looking diagonally, this seems to be an artifact of this: https://github.com/fog/fog/commit/c23eb01c8d57b295ef2cad0ba6aa8c23f79aa7a8

Theorising here: it was probably conceived as an easier way to set disk_type (without resorting to `get_disk_type` first), but probably was dismissed when the author saw that disk_types are not equally distributed between the zones.

BTW I wouldn't accept an autofix like that, it makes the code even more unreadable IMO :/